doc/project/tsc.rst

```diff
@@ -0,0 +1,218 @@
+  document.
+- The number of elected members shall not exceed 20% of the total of appointed
+  members.
+- Existing TSC members who were elected before May 2024 shall be re-confirmed
+  after completing the 2 year term since they were first elected.
+
+Suspensions
++++++++++++
+
+As noted under Section 8b of the Project Charter, Voting rights for a
+representative who misses three consecutive meetings are subject to suspension
+and suspended representatives do not count towards the quorum requirement.
+
+A representative’s suspension will end and voting rights be restored at the start
+of the next attended meeting. The TSC enforces the suspension policy for voting
+members who miss three consecutive TSC weekly meetings.
+
+Multi-day meetings (F2F events) are counted as "one" meeting. The TSC voted on
+February 16, 2022 to discontinue default enforcement of the suspension policy.
+The TSC voted on January 18, 2023 to re-enact default enforcement of the
+suspension policy.
+
+Notice of suspension will be sent to representatives who miss three consecutive
+meetings, noting that rights will be restored upon next attendance of a TSC
+meeting.
+
+.. note::
+
+   As per Section 4b of the Project Charter, Platinum and Silver
+   Members may choose to opt out of a voting seat on the TSC.
+
+   Members who opt out and then wish to reclaim their seat later will have their
+   voting rights restored at the start of the second consecutive meeting
+   attended following notification to the TSC Chair.
+
+
+Voting
+++++++
+
+Voting in the Zephyr Project is defined under Section 8 of the Project Charter.
+
+Additional points of clarity / TSC interpretation have been added below:
+
+- The Governing Board may opt to update the Charter to include the below
+  refinements. Until then, additional clarifications (if/where needed) will be
+  discussed in the Process Working Group, and approved in the TSC.
+- TSC In-Meeting Voting For items requesting an in-meeting vote of the Zephyr
+  Technical Steering Committee (TSC), assuming quorum requirements have been
+  met, the default voting mechanism will be a verbal motion to determine if
+  there is general consensus.
+- If there are no objections to a motion being brought forward, general
+  consensus is assumed and the motion passes.
+- Should there be any objections raised, the vote will move to email, and be
+  executed using the Voting Guidelines outlined in Section 8 of the Project
+  Charter.
+- Should a motion be deemed urgent by the TSC Chair, and assuming quorum
+  requirements have been met, the Chair may call for a roll call vote
+  in-meeting.
+
+Voting Options
+--------------
+
+- Voting Options are:
+
+  - "Yes",
+  - "No" or
+  - "Abstain"
+
+Abstention
+----------
+
+ - Abstentions do not count in tallying the vote negatively or positively; when
+   members abstain, they are in effect attending only to contribute to a quorum.
+ - Abstentions count toward Quorum (see present or participating
```

Comment:
Redundant vs "when members abstain, they are in effect attending only to contribute to a quorum" ?

Reply:
ok, removed duplicate part